app/src/main/AndroidManifest.xml

```diff
@@ -46,8 +46,16 @@
     <uses-permission android:name="android.permission.USE_BIOMETRIC" />
 
 
+    <!-- This works fine prior to Android version 11 -->
     <uses-permission android:name="android.permission.QUERY_ALL_PACKAGES" tools:ignore="QueryAllPackagesPermission"/>
 
+    <queries>
+        <intent>
+            <action android:name="android.intent.action.ALL_APPS"/>
```

Comment:
I think this is actually not helpful here. Correct me if I'm wrong, but I think this could be removed.